app/MMB/src/components/menu/MenuItem.js

```diff
@@ -30,9 +30,12 @@ type Props = {|
  * `------------------------------`
  */
 export default function MenuItem(props: Props) {
+  const styleSheet = createStyleSheet(props);
```

Comment:
Isn't this really just the same as using inline styles? You create a new object on each render

Reply:
What do you propose? I have to create a new object on each render because this object depends on props.

Reply:
@tbergq Updated - please check again.